XOutput/UI/Windows/MainWindowViewModel.cs

```diff
@@ -147,6 +144,13 @@ public void Initialize(Action<string> log)
             Model.Settings = settings;
             RefreshGameControllers();
 
+            if (!settings.DisableAutoRefresh)
+            {
+                timer.Interval = TimeSpan.FromMilliseconds(5000);
+                timer.Tick += (object sender1, EventArgs e1) => { RefreshGameControllers(); };
+                timer.Start();
+            }
+
```

Comment:
This way the timer will not be started even if you click it.
I think the proper way would to change from
`(object sender1, EventArgs e1) => { RefreshGameControllers(); }`
to 
`(object sender1, EventArgs e1) => { if (!settings.DisableAutoRefresh) { RefreshGameControllers(); } }`

Reply:
Yes, that makes more sense. I have changed the timer logic.